homeassistant/components/tuya/select.py

```diff
@@ -274,6 +274,24 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Air Purifier
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf46h2s6dzm
+    "kj": (
+        SelectEntityDescription(
+            key=DPCode.COUNTDOWN,
+            name="Countdown",
+            device_class=TuyaDeviceClass.COUNTDOWN_MINUTES,
+            entity_category=EntityCategory.CONFIG,
+            icon="mdi:timer-cog-outline",
+        ),
+        SelectEntityDescription(
+            key=DPCode.COUNTDOWN_SET,
+            name="Countdown",
+            device_class=TuyaDeviceClass.COUNTDOWN_HOURS,
```

Comment:
I think we should just name this:

```suggestion
            device_class=TuyaDeviceClass.COUNTDOWN,
```